packages/frontend-template/playwright-report/index.html

```diff
@@ -7,56 +7,62 @@
     <meta name='color-scheme' content='dark light'>
     <meta name='viewport' content='width=device-width, initial-scale=1.0'>
```

Comment:
Why is this file changed?

Reply:
Oh sorry , i think this was a mistake and was changed while i started the tests.

Reply:
please revert those changes